skopt/acquisition.py

```diff
@@ -105,10 +105,12 @@ def gaussian_lcb(X, model, kappa=1.96, return_grad=False):
         It should have a ``return_std`` parameter that returns the standard
         deviation.
 
-    * `kappa`: [float, default 1.96]:
+    * `kappa`: [float, default 1.96 or 'variance']:
         Controls how much of the variance in the predicted values should be
         taken into account. If set to be very high, then we are favouring
         exploration over exploitation and vice versa.
+        If set to 'variance', the acquisition function will only use the variance
```

Comment:
Do you talk about the name of acquisition function? Some might have weird associations with 'std' as abbreviation :sweat_smile: